packages/dds/merge-tree/api-report/merge-tree.legacy.alpha.api.md

```diff
@@ -87,104 +78,6 @@ export abstract class BaseSegment implements ISegment {
     wasMovedOnInsert?: boolean | undefined;
 }
 
-// @alpha @deprecated (undocumented)
-export class Client extends TypedEventEmitter<IClientEvents> {
```

Comment:
@tylerbutler  do you know why these type removals don't break the type tests? i would expect those to need suppression, but they don't for some reason.

Reply:
i guess it makes sense if the look at the imports 
```
import type * as old from "@fluidframework/merge-tree-previous/internal";

import type * as current from "../../index.js";
```

as the types still exist off internal, just not legacy, so are still available off `import type * as current from "../../index.js";
`